secantTests/Util/RecoveryPhraseStorageTests.swift

```diff
@@ -0,0 +1,228 @@
+//
+//  RecoveryPhraseStorageTests.swift
+//  secantTests
+//
+//  Created by Lukáš Korba on 10.03.2022.
+//
+
+import XCTest
+@testable import secant_testnet
+
+extension KeyStoringError {
+    var debugValue: String {
+        switch self {
+        case .alreadyImported: return "alreadyImported"
+        case .uninitializedWallet: return "uninitializedWallet"
+        case .storageError: return "storageError"
+        }
+    }
+}
+
+class RecoveryPhraseStorageTests: XCTestCase {
+    // MARK: - Recovery Phrase tests
```

Comment:
This seems pretty redundant and not needed

Reply:
done